_source/_posts/2019-05-31-simple-auth-express-fifteen-minutes.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: blog_post
-title: 'Build Simple Authentication in Express in 15 Minutes'
+title: "Build Simple Authentication in Express in 15 Minutes"
 author: braden-kelley
 by: contractor
 communities: [javascript]
-description: 'A tutorial on building simple authentication in Express.'
+description: "A tutorial on building simple authentication in Express."
 tags: [node, nodejs, express, authentication]
 tweets:
-  - 'Learn to build simple authentication in #ExpressJS!'
-  - 'Want to build authentication in #ExpressJS? Check this out!'
+  - "Learn to build simple authentication in #ExpressJS!"
+  - "Want to build authentication in #ExpressJS? Check this out!"
   - "Need to learn how to build simple authentication in #ExpressJS? We've got you covered!"
 image: blog/featured/okta-node-tile-books-mouse.jpg
 type: conversion
 changelog:
   - 2020-06-15: Updated to use the v4.0.1 version of  Okta Middleware. See the code changes in the [example app on GitHub](https://github.com/oktadeveloper/okta-node-express-15-minute-auth-example/pull/6). Changes to this article can be viewed in [oktadeveloper/okta-blog#325](https://github.com/oktadeveloper/okta-blog/pull/325).
+  - 2021-10-18: Updated to use latest Okta dashboard screenshots and latest Okta libraries.
```

Comment:
We should link to this PR so people can see what changed. Similar to the changelog above this one. Also, the changelog ordering is descending, so newest update should be first.